upload-requirements.txt

```diff
@@ -1 +1 @@
-twine==1.9.1
+twine>=1.9.1
```

Comment:
Could you amend to revert this change?
Aggressively upgrading `twine` has actually bitten us in the past. It's pegged at a specific version intentionally (and yes, I know that's a touch weird).

Otherwise, I'll revert in the merge.

Reply:
I can do that, what about setting it to `==1.11.0` ? It's been released in March and quite stable, with important updates (see https://twine.readthedocs.io/en/latest/changelog.html ).